src/js/tabs/pid_tuning.js

```diff
@@ -579,6 +586,7 @@ TABS.pid_tuning.initialize = function (callback) {
                     $('#pid_main .pid_titlebar2 th').attr('colspan', 5);
                     $('.derivativeText').text(i18n.getMessage("pidTuningDerivative"));
                 }
+                TuningSliders.updatePidSlidersDisplay();
```

Comment:
This will fix legacy sliders
```suggestion
                if (semver.gte(FC.CONFIG.apiVersion, API_VERSION_1_44)) {
                    TuningSliders.updatePidSlidersDisplay();
                }
```